python/packages/ai/teams/ai/planner/openai_planner.py

```diff
@@ -88,7 +84,11 @@ async def generate_plan(
             if assistant_prefix:
                 # The model sometimes predicts additional text
                 # for the human side of things so skip that.
-                position = result.lower().index(assistant_prefix.lower())
+                try:
+                    position = result.lower().index(assistant_prefix.lower())
+                except ValueError:
```

Comment:
I have fixed this by using `find()` function in my PR, which is shorter than try/catch block. You can merge my changes after my PR is merged.